tests/Nox.Types.Tests/Types/Nuid/NuidTests.cs

```diff
@@ -1,11 +1,31 @@
 ﻿// ReSharper disable once CheckNamespace
+using FluentAssertions;
+
 namespace Nox.Types.Tests.Types;
 
 public class NuidTests
 {
+    private const string TestStringValue = "!#123TestValue456!#";
+    private const uint ExpectedNuidValue = 598674021;
+    private const string ExpectedBase36 = "XTNFW9";
+    private readonly Guid ExpectedGuid = new("00000000-0000-0000-0000-000023af0a65");
+
     [Fact]
-    public void When_Create_Should()
+    public void CreateNuiidFromString_CheckImplementation_ShouldNotBeChanged()
```

Comment:
```suggestion
    public void FromString_Construction_ShouldReturnExpected()
```

Reply:
IMO this test implementation is not what we want to validate, because if we introduce an issue, this test will always be valid